k8s/namespaces/wa/wa-task-management-api/aat.yaml

```diff
@@ -7,3 +7,5 @@ spec:
     java:
       environment:
         LOGGING_LEVEL_UK_GOV_HMCTS: DEBUG
+        ROLE_ASSIGNMENT_URL: "http://am-role-assignment-service-pr-1096.service.core-compute-aat.internal"
```

Comment:
is this URL correct? I'd expect the pr URL will disappear with the PR

Reply:
this PR Role assignment have set it for by passing validation and it get updated everyday.